Macontainer/ContentView.swift

```diff
@@ -392,27 +393,8 @@ struct ContentView: View {
             }
             .toolbar {
                 HStack {
-                    if viewModel.hasNewerVersion {
-                        Button(action: {
-                            if let url = URL(string: "https://github.com/apple/container/releases")
-                            {
-                                NSWorkspace.shared.open(url)
-                            }
-                        }) {
-                            Text(viewModel.cliVersion ?? "Unknown version")
-                                .foregroundColor(.orange)
-                                .underline()
-                        }
-                        .buttonStyle(PlainButtonStyle())
-                        .help(
-                            viewModel.latestVersion != nil
-                                ? "Update available: \(viewModel.latestVersion!) - Click to view releases"
-                                : "Update available - Click to view releases")
-                    } else {
-                        Text(viewModel.cliVersion ?? "Unknown version")
-                            .foregroundColor(.secondary)
-                            .help("Version is up to date")
-                    }
+                    Text(viewModel.cliVersion ?? "Unknown version")
+                        .foregroundColor(.secondary)
```

Comment:
Leave this code as it was, don't touch it. That's about a newer version of the CLI available, not this app.